log4j-transform-maven-plugin/src/main/java/org/apache/logging/log4j/transform/maven/LocationMojo.java

```diff
@@ -189,11 +192,17 @@ private WrappedIOException(IOException cause) {
         }
     }
 
-    private void validateLog4jVersion() throws MojoExecutionException {
-        Artifact log4jApi = project.getArtifacts().stream()
+    private boolean validateLog4jVersion() throws MojoExecutionException {
```

Comment:
This method should probably either throw or return a `boolean`. Can you replace the `throw` statements with log + return?

Reply:
If the version of Log4j API is too old, we can log an error and skip the execution.